apps/src/labs/types.ts

```diff
@@ -96,6 +97,11 @@ export interface LevelData {
   startSources: Source;
 }
 
+export interface LevelVideo {
```

Comment:
I guess ideally this would be a variant type of level data.  To that end, I've prepped a standalone PR (which builds on this one) which shows one idea for that.  Would love thoughts: https://github.com/code-dot-org/code-dot-org/pull/52570

Reply:
Merged into this PR.